Scripts/debug/debug_blast_parser.py

```diff
@@ -246,7 +246,7 @@ def test_blast_output(outfile):
     if len(args) != 1:
         sys.stderr.write(USAGE)
         sys.exit(-1)
-    TESTFILE, = args
+    (TESTFILE,) = args
```

Comment:
`TESTFILE = args[0]`?

Reply:
This dates back to the original check in, bbe8f5c94839aeed927ab65afca485240c33d304 - but I agree with the suggestion.

Reply:
However, the script no longer works - ``Bio.ParserSupport`` was removed in Release 1.72, while ``Bio.Blast.NCBIStandalone`` was moved away to ``Bio.SearchIO._legacy`` as an interim measure. We can just remove the entire script.